sqlglot/dialects/mysql.py

```diff
@@ -717,3 +730,9 @@ def _oldstyle_limit_sql(self, expression: exp.Show) -> str:
                 limit_offset = f"{offset}, {limit}" if offset else limit
                 return f" LIMIT {limit_offset}"
             return ""
+
+        def chr_sql(self, expression: exp.Chr) -> str:
+            this = self.expressions(sqls=[expression.this] + expression.expressions)
```

Comment:
Isn't sqls` supposed to be a list of strings?

Reply:
i think the type hint is wrong in expressions, it can be exp or str